SECURITY_GUIDANCE.md

```diff
@@ -223,6 +246,11 @@ These settings can passed as [user data](https://docs.aws.amazon.com/AWSEC2/late
 They apply to any Bottlerocket variant.
```

Comment:
Does this comment still apply if you're talking about segmenting this setting's default by variant?

Reply:
It's a correct statement today, and would be a no-op if we change the default.